pyaedt/modeler/Primitives.py

```diff
@@ -1304,9 +1304,9 @@ def create_region(self, pad_percent=300):
 
         Parameters
         ----------
-        pad_percent : float or list of floats, optional
+        pad_percent : float or list of floats or list of str, optional
```

Comment:
I thought when different data formats were acceptable, you listed them with commas between the names and not "or"--so: "float, list of floats, list of str".